sdk/keyvault/azure_security_keyvault_keys/src/main.rs

```diff
@@ -0,0 +1,3 @@
+fn main() {
```

Comment:
Delete this file?

Reply:
Definitely. This will create an executable crate.